Problem1/even_numbers_adder.rb

```diff
@@ -2,8 +2,6 @@
 
 class EvenNumbersAdder
   def add(numbers)	
-    sum = numbers.find_all(&:even?).reduce(:+)
-		
-    return sum == nil ? 0 : sum
+    numbers.find_all(&:even?).reduce(:+).to_i
```

Comment:
It is weird that you are doing `to_i` since all that you are operating with are integers already.
`to_i` is there just in the case there are no even numbers, so the sum is `nil`

I was thinking at something like:
`numbers.find_all(&:even?).reduce(:+) || 0`